CMakeLists.txt

```diff
@@ -276,7 +276,7 @@ if(NOT DEFINED USE_VULKAN)
       "ANDROID" OFF)
 endif()
 
-option(USE_SLEEF_FOR_ARM_VEC256 "Use sleef for arm" OFF)
+option(USE_SLEEF_FOR_ARM_VEC256 "Use sleef for arm" ON)
```

Comment:
Shouldn't this be a conditional option on ARM cpu?

Reply:
Good question. Maybe it's better to leave USE_SLEEF_FOR_ARM_VEC256 default off, and use another flag for Apple silicon. I will make some change accordingly.

Reply:
Hi @malfet I made some change to enable sleef on macOS with Apple silicon only, other ARM cpus will not be impacted. Do you think it's ok to merge? If so, can you merge it? I don't have the permission. There are still several test failures on linux, which seem unrelated to this change.